src/AveragerTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.Test;
+
+import static java.lang.Double.NaN;
+import static org.junit.jupiter.api.Assertions.*;
+
+class AveragerTest {
+
+    @Test
+    void intergerify() {
+         String[] input = {"4","5","6","7"};
+         int[] expected = {4,5,6,7};
+         assertArrayEquals(expected,Averager.intergerify(input));
```

Comment:
For readability, please put a space between your params:
`assertArrayEquals(expected, Averager.intergerify(input));`